src/components/homesections/homesections.js

```diff
@@ -392,28 +392,31 @@ import ServerConnections from '../ServerConnections';
         };
     }
 
-    function getContinueWatchingItemsHtml(items) {
-        const cardLayout = false;
-        return cardBuilder.getCardsHtml({
-            items: items,
-            preferThumb: true,
-            shape: getThumbShape(),
-            overlayText: false,
-            showTitle: true,
-            showParentTitle: true,
-            lazy: true,
-            showDetailsMenu: true,
-            overlayPlayButton: true,
-            context: 'home',
-            centerText: !cardLayout,
-            allowBottomPadding: false,
-            cardLayout: cardLayout,
-            showYear: true,
-            lines: 2
-        });
+    function getContinueWatchingItemsHtmlFn(useEpisodeImages) {
+        return function (items) {
+            const cardLayout = false;
+            return cardBuilder.getCardsHtml({
+                items: items,
+                preferThumb: true,
+                inheritThumb: !useEpisodeImages,
+                shape: getThumbShape(),
+                overlayText: false,
+                showTitle: true,
+                showParentTitle: true,
+                lazy: true,
+                showDetailsMenu: true,
+                overlayPlayButton: true,
+                context: 'home',
+                centerText: !cardLayout,
+                allowBottomPadding: false,
+                cardLayout: cardLayout,
+                showYear: true,
+                lines: 2
+            });
+        };
     }
 
-    function loadResumeVideo(elem, apiClient) {
+    function loadResumeVideo(elem, apiClient, user, userSettings) {
```

Comment:
`user` seems to be unused?

Reply:
It's unused, but all the other `load*` functions had similar signatures (see line 134), that's why I added that in. Can easily remove it if that's not a concern.